### PR TITLE
Add Todoist Box

### PR DESCRIPTION
**Pull request by Justgonc**
Added Todoist Box to the list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,9 @@
 
 Displaying data from external services in a pinned gist.
 
-- [bird-box](https://github.com/matchai/bird-box) - Update a pinned gist to contain the latest tweets of a Twitter user.
+- [bird-box](https://github.com/matchai/bird-box) - Update a gist to contain your latest tweetUpdate a pinned gist to contain the latest tweets of a Twitter user.
```
Comment (nathanielchi) [inline README.md:15]: Please fix this. There is some extra lines on line 15

**Review by nathanielchi — CHANGES_REQUESTED** [review]
Thanks @Justgonc! This looks great, However it does have one error in it. Please fix that so I can merge it.

File: README.md (hunk 1 above)
Comment (Justgonc) [reply README.md]: Fixed it, Sorry about that.

**Review by nathanielchi — APPROVED** [review]
LGTM

**Pull request merged**